README.md

```diff
@@ -147,6 +147,8 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [SQUANCH](https://github.com/att-innovate/squanch) - Distributed simulation framework for quantum networks and channels.
 - [QuNetSim](https://github.com/tqsd/QuNetSim) - Quantum network simulation framework.
 - [The Walrus](https://github.com/xanaduAI/thewalrus)- [Xanadu](https://www.xanadu.ai)'s library for simulating Gaussian Boson Sampling.
+- [gdsfactory](https://gdsfactory.github.io/gdsfactory/) - Open Source Python library designed for crafting chips (Photonics, Analog, Quantum, MEMs, and more), 3D printed objects, and PCBs.
+- [gdsfactory plugins](https://gdsfactory.github.io/gplugins) - Plugins for Simulating Analog, RF and Photonics circuits.
```

Comment:
It looks like the latter is just a set of plugins for the former? If it's the same project, it should be on one line.